peachjam/views.py

```diff
@@ -1,16 +0,0 @@
-from django.contrib.auth.mixins import LoginRequiredMixin, PermissionRequiredMixin
-from django.views.generic import TemplateView
-
-
-class AuthedViewMixin(LoginRequiredMixin, PermissionRequiredMixin):
```

Comment:
Where have these moved to?

Reply:
`AuthedViewMixin` class is not in use anywhere that I can see. The Homepage class is in home.py